tests/lib/screenshot-testing/support/test-environment.js

```diff
@@ -34,6 +35,33 @@ TestingEnvironment.prototype.reload = function () {
     }
 };
 
+/**
+ * Overrides a config entry.
+ *
+ * You can use this method either to set one specific config value `overrideConfig(group, name, value)`
+ * or you can set a whole group of values `overrideConfig(group, valueObject)`.
+ */
+TestingEnvironment.prototype.overrideConfig = function (group, name, value) {
```

Comment:
this method should be ideally used to override configs in the future in UI tests